### ISSUE TITLE
Treat `specify` and `it` identically when checking RSpec style

### ISSUE THREAD

**Issue by scottmatthewman** [altmetric/house_style#8]
Implementing `house_style` on an internal project, I noticed that the following line triggered a warning in our configured `Style/BlockDelimiters` cop

```ruby
specify { should_not render_template(:show) }
```

Changing to `it` did not trigger the warning:

```ruby
it { should_not render_template(:show) }
```

`it` is in the cop's `IgnoredMethods` collection, along with `lambda` and `proc`. `specify` should be treated the same.
